src/workspaces/dashboard/OwnerNotice.ts

```diff
@@ -20,7 +20,7 @@ export const OwnerNotice = (props: OwnerNoticeProps): ReactNode => {
   const { workspace } = props;
   const { owners = [], accessLevel } = workspace;
 
-  const [acl, setAcl] = useState<WorkspaceAcl>();
+  const [acl, setAcl] = useState<RawWorkspaceAcl>();
```

Comment:
This type was wrong-- it's the raw result from Rawls, not the cleaned up version.